src/input/base-input.js

```diff
@@ -184,13 +184,15 @@ class BaseInput<T: EventTarget> extends React.Component<
     }[this.props.size];
     return (
       <MaskToggleButton
-        aria-label={label}
```

Comment:
should `aria-label` be maintained?

Reply:
oops - i forgot to add these props back in after trying something